### ISSUE TITLE
1. not connfirmed pay - 2. without any notice contract was immediately created

### ISSUE THREAD

**Issue by vrfebet** [ethereum/mist#3939]
```
Ethereum Wallet Version: `0.10.0`
OS & Version: windows10 64
Node version: `geth 1.8.10`
Number of blocks synchronized: 0
```

I have two ques:
1) i send test payment, but in not 14 days connfirmed
![eth-not-test-send](https://user-images.githubusercontent.com/23390160/40889534-dc05933c-6757-11e8-90e2-4454553c90a0.png)
what with this?

2) I like built "wallet contract" in "Etherbase" ...
I clicked to "ADD WALLET CONTRACT" - without any notice, a  unnamed contract was immediately created (many days are being created)
i don't now  parametrs of contract now...
![wallet contracts-2](https://user-images.githubusercontent.com/23390160/40889562-28e94446-6758-11e8-844e-dfe8a7132986.png)

I do not know if this is related:
UI:
2a3fe1c….js:340 Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at Object.Helpers.getENSName (2a3fe1c….js:340)
    at 2a3fe1c….js:319
    at Array.forEach (<anonymous>)
    at Function.A.each.A.forEach (2a3fe1c….js:1)
    at updateBalances (2a3fe1c….js:319)
    at observeLatestBlocks (2a3fe1c….js:319)
    at connectToNode (2a3fe1c….js:336)
    at 2a3fe1c….js:377
    at 2a3fe1c….js:187

what with this?
many times I thank a developer who will be willing to help because otherwise the wallet is unusable

PeGoS

**Comment by vrfebet** [comment]
![eth-not-send10](https://user-images.githubusercontent.com/23390160/41356277-18c08fda-6f13-11e8-8369-10472add71f6.png)

Are not we really interested in our problems?
This is the official wallet of the billions of currency?

**Comment by vrfebet** [comment]
![eth-not-send11](https://user-images.githubusercontent.com/23390160/42048840-e6a4cf60-7af3-11e8-8267-f8cac0e197ad.png)
there is nothing to add
the luck I used  for the testing  only 0.3 eth

**Comment by ryanio** [comment]
Hey, we released a new version (`0.11.0`) with better node connection features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how
we can help. Thanks!
